File: src/OpenSilver.Showcase/Other/Internals/Menu/PageInfo.vb
Imports System.Windows.Media

Namespace OpenSilver.Showcase
    Public Class PageInfo
        Implements IMenuElement

        Public Property Name As String
        Public Property Path As String
        Public Property Icon As String
        Public Property IconBrush As Brush
        Public Property IsVisibleInMenu As Boolean = True
    End Class
End Namespace

File: src/OpenSilver.Showcase/Other/Internals/Menu/PageInfo.fs
namespace OpenSilver.Showcase

open System.Windows.Media

type PageInfo() =
    interface IMenuElement
    member val Name: string = null with get, set
    member val Path: string = null with get, set
    member val Icon: string = null with get, set
    member val IconBrush: Brush = null with get, set
    member val IsVisibleInMenu: bool = true with get, set
